airflow/dags/reporting_database_generation/generate-reporting-db.py

```diff
@@ -49,43 +47,63 @@ class SQLTemplatedPythonOperator(PythonOperator):
     'retries': 0,
     'retry_delay': timedelta(minutes=5),
     'concurrency': 4,
-    'queue': 'disambiguator'
-    # 'pool': 'backfill',
-    # 'priority_weight': 10,
-    # 'end_date': datetime(2016, 1, 1),
+    'queue': 'data_collector'
 }
 
 # REPORTING DB
 
 reporting_db_dag = DAG("reporting_database_generation"
                        , default_args=default_args
-                       , start_date=datetime(2022, 6, 30)
-                       , schedule_interval=None
+                       , start_date=datetime(2022, 1, 1)
+                       , end_date=datetime(2022, 7, 1)
```

Comment:
Wouldn't this end date prevent it from running at all v/

Reply:
This is just for testing!